### DISCUSSION TITLE
Material Sample form not displaying

### DISCUSSION THREAD

**Discussion by ricklevy21** [Symbiota/community-board#130] [Problems/Bugs]
A colleague and myself seem to be having an issue with the new Material Sample tool, where the form does not display. Other user accounts linked to our collection are able to see and use the form, and I am needless to say very jealous. Screen shot attached. When I click the yellow plus sign, a "#" is added to the end of the URL, but noting happens.
![materialSample_tab](https://user-images.githubusercontent.com/11430669/159800822-bdcdca93-b485-4c4f-96ce-a7e31dab6028.jpg)

**Comment by themerekat** [comment]
How odd! I'll see what the team thinks.
